migrations/026_sub_flair_display_order.py

```diff
@@ -0,0 +1,19 @@
+"""Peewee migrations -- sub_flair_display_order.py.
+Adds a new field to the sub_flair model to allow sub mods to choose the order.
+"""
+
+import peewee as pw
+
+SQL = pw.SQL
+
+
+def migrate(migrator, database, fake=False, **kwargs):
+    """Write your migrations here."""
+    migrator.add_fields(
+        migrator.orm['sub_flair'],
+        display_order=pw.IntegerField(constraints=[SQL("DEFAULT 1")]),
```

Comment:
This shouild be like this:

```suggestion
        display_order=pw.IntegerField(default=1),
```

Otherwise migrations will fail with `ValueError: display_order is not null but has no default`